pages/prompt-input/simple.page.tsx

```diff
@@ -191,6 +203,31 @@ export default function PromptInputPage() {
                   warning={hasWarning}
                   ref={ref}
                   disableSecondaryActionsPaddings={true}
+                  customPrimaryAction={
+                    hasPrimaryActions ? (
+                      <Box padding={{ right: 'xxs', top: 'xs' }}>
+                        <ButtonGroup
```

Comment:
The focus outline for custom primary actions looks different compared to the default one. It is in a close proximity to the area's border. Is that expected?

<img width="161" height="139" alt="Image" src="https://github.com/user-attachments/assets/810790e8-1d07-4a91-b869-9547a7dfb9fc" />

Reply:
nit: the custom actions do not have any associated effect, while the original action does trigger an alert

Reply:
(however, updated styling to make it simpler and somewhat more consistent)